src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs

```diff
@@ -240,7 +240,7 @@ public bool TryFetch(LocalUserRegistryOptionPersister persister, OptionKey2 opti
         {"dotnet_provide_date_and_time_completions", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.ProvideDateAndTimeCompletions")},
         {"dotnet_log_telemetry_for_background_analyzer_execution", new FeatureFlagStorage(@"Roslyn.LogTelemetryForBackgroundAnalyzerExecution")},
         {"dotnet_lightbulb_skip_executing_deprioritized_analyzers", new FeatureFlagStorage(@"Roslyn.LightbulbSkipExecutingDeprioritizedAnalyzers")},
-        {"dotnet_enable_lsp_pull_diagnostics", new FeatureFlagStorage(@"Lsp.PullDiagnostics")},
+        {"dotnet_enable_pull_diagnostics", new FeatureFlagStorage(@"Roslyn.EnablePullDiagnostics")},
```

Comment:
`Lsp.PullDiagnostics` cannot be changed - it's the value we use in the targeted notification payload:
```
"FeatureFlags": {
    "Lsp.PullDiagnostics": 1
},
```

Reply:
reverted.